live_site.py

```diff
@@ -0,0 +1,433 @@
+			droplet_exists = x
+			return droplet_exists
+	else:
+		return None
+
+# Deletes all droplets
+def del_all_droplets():
+	my_droplets = get_all_droplets()
+
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			x = details['name']
+			dropletId = details['id']
+			del_droplet(dropletId)
+	else:
+		click.secho('All droplets have been deleted!', fg='bright_green')
+
+
+def create_droplet(droplet):
+
+	droplet = digitalocean.Droplet(token=api_token,
+								  name=droplet,
+								  region='nyc1',
+								  image='ubuntu-20-04-x64',
+								  size_slug='s-2vcpu-4gb', 
+								  backups=True, 
+								  ipv6=False, 
+								  ssh_keys=[28355743, 29166498], #white_key & carrot_key
+								  user_data='',
+								  private_networking='',
+								  volumes=[],
+								  tags="web")
+	droplet.create()
+	d = droplet.__str__()      
+	click.secho(">>> New droplet has been created with a Droplet ID and Droplet Name: ")
+	click.secho(d,  fg='bright_green')      
+
+
+def get_droplet_id():
+	my_droplets = get_all_droplets()
+
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			x = details['name']
+			dropletId = details['id']
+			return dropletId
+	else:
+		return None		
+
+
+def get_hostname_ip(droplet):
+# This function is to cycle through all the droplets and 
+# get the IP Address of the droplet.
+# Also checks if the droplet is up, thus returns the IP Address.  
+	result = ''
+	ip = ''
+
+	click.secho(">>> Inspecting droplets...", fg='bright_yellow')   
+	my_droplets = get_all_droplets()	
+	
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			if details['name'] == droplet:
+				#click.echo(details['name'])                
+				if key == 'networks':
+					result = value      
+					for key, value in result.items():
+						if key == 'v4':
+							v4 = value[1]
+							ip = v4['ip_address']
+							#click.echo(ip)
+							return ip
+	else:
+		click.secho(">>> Your droplet {}' doesn't exist!".format(droplet), fg='bright_red')
+		click.secho(">>> We can't add your data at the moment.", fg='bright_red')                     
+
+
+def get_all_domains():
+
+	api_url = '{0}domains'.format(api_url_base)
+
+	response = requests.get(api_url, headers=headers)
+
+	if response.status_code == 200:
+		return json.loads(response.content.decode('utf-8'))
+	else:
+		return None
+
+
+def add_new_domain(domain_name, ip_address):
+
+	api_url = '{0}domains'.format(api_url_base)    
+
+	domain = {'name': domain_name, 'ip_address': ip_address}
+
+	response = requests.post(api_url, headers=headers, json=domain)
+
+	if response.status_code == 201:
+		domain = json.loads(response.content)
+		return domain
+	else:
+		click.secho('[?] Unexpected Error: [HTTP {0}]: Content: {1}'.format(response.status_code, response.content), fg='bright_red')
+		return None
+			
+
+def check_domain(domain_name):
```

Comment:
I would challenge you to take a function like def check_domain(domain_name): and refactor it (refactoring simply means rearranging code, see: https://en.wikipedia.org/wiki/Code_refactoring) to work as a gate test